source/install/native.rst

```diff
@@ -22,23 +22,39 @@ Install QIIME 2 within a ``conda`` environment
 
 Once you have Miniconda installed, create a ``conda`` environment and install the QIIME 2 Core 2017.6 distribution within the environment. We **highly** recommend creating a *new* environment specifically for the QIIME 2 release being installed, as there are many required dependencies that you may not want added to an existing environment. You can choose whatever name you'd like for the environment. In this example, we'll name the environment ``qiime2-2017.6`` to indicate what QIIME 2 release is installed (i.e. ``2017.6``).
 
-The installation process differs slightly across platforms. Please choose the installation instructions that are appropriate for your platform.
 
-macOS/OS X (64-bit)
-~~~~~~~~~~~~~~~~~~~
+.. raw:: html
+
+   <style>
```

Comment:
Can you move this CSS into the [stylesheet](https://github.com/qiime2/docs/blob/master/source/theme/static/style.css)? Perhaps you can refactor the [`download` directives](https://github.com/qiime2/docs/blob/master/source/theme/static/style.css#L92-L99) to share the same styles? Also, there is some extra padding at the bottom, can you make sure that gets removed (hint: reusing the existing styles should help with that)?